index.js

```diff
@@ -2,30 +2,57 @@
  * Q0: Write a function called `countdown` that counts down from 10 on the console and
  * then logs "Blast Off!"
  */
-
+function blastOff() {
+	for (count = 10; count > 0; count--) {
+		console.log(count);
+	};
+	console.log("Blast Off!");
+};
 
 /**
  * Q1: Write a function called `max` that has two Number parameters and returns
  * the larger of the two.
  */
-
+function max(x, y) {
+	if (x > y) {
+		return x;
+	} else {
+		return y;
+	};
+};
 
 /**
  * Q2: Write a function called `longer` that has two String parameters and returns
  * the String that is longer.
  */
-
+function longer(stringOne, stringTwo) {
+	if (stringOne.length > stringTwo.length) {
+		return stringOne;
+	} else {
+		return stringTwo;
+	};
+};
 
 /**
  * Q3: Write a function called `isEven` that has one Number parameter and returns `true` if
  * if the number is even and `false` otherwise.
  */
+function isEven(number) {
+	if (number % 2 === 0) {
+		return true;
+	} else {
+		return false;
+	};
+};
```

Comment:
Nice work on all of these so far! Here is an example of another, slightly more succinct way to write the same thing:

```js
function isEven(number) {
    return number % 2 === 0;
}
```

Since the result of an equality is itself a boolean, we can return that directly.